### PR TITLE
Typo fix and other changes

### LINKED ISSUE #106: "miliseconds" typo
Noticed a small typo I've found in the definitions of `TIME_UNITS`: "miliseconds" should be "milliseconds"

### PR DESCRIPTION
**Pull request by harshpalan**
Hi, creating a PR to fix #106, #97 and the tracks->clips typo fix.

### CODE REVIEW

File: soundata/datasets/tau2021sse_nigens.py [hunk 1]
```diff
@@ -359,14 +359,14 @@ def validate_locations(locations):
 
 
 class Clip(core.Clip):
-    """TAU NIGENS SSE 2021 Track class
+    """TAU NIGENS SSE 2021 clip class
```
Comment (ghost) [inline soundata/datasets/tau2021sse_nigens.py]: maybe writing `Clip` here (and in the rest of the updated loaders? I mean, using capital C?

**Review by ghost — APPROVED** [review]

**Comment by ghost** [comment]
hey @harshpalan, I am just leaving a tiny comment here but looks good :)

File: soundata/datasets/tau2021sse_nigens.py (hunk 1 above)
Comment (harshpalan) [reply soundata/datasets/tau2021sse_nigens.py]: Sure, will make the changes.

**Comment by harshpalan** [comment]
@genisplajai changes are made, can you please review it? Thanks.

**Review by genisplaja — APPROVED** [review]
:)

**Review by genisplaja — APPROVED** [review]
:)

**Review by genisplaja — APPROVED** [review]
:)

**Pull request merged**